thirdeye/thirdeye-frontend/app/pods/components/composite-anomalies/parent-anomalies/component.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Parent Anomalies Component
+ *
+ * Display a table containing composite anomalies
+ * @module composite-anomalies/parent-anomalies
+ * @property {string} title   - Heading to use on the table
+ * @property {object[]} data  - [required] array of composite anomalies objects
+ *
+ * @example
+ * {{composite-anomalies/parent-anomalies title=<title> data=<data>}}
+ *
+ * @exports composite-anomalies/parent-anomalies
+ */
+
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { A as EmberArray } from '@ember/array';
+
+import moment from 'moment';
+import config from 'thirdeye-frontend/config/environment';
+import * as anomalyUtil from 'thirdeye-frontend/utils/anomaly';
+
+const TABLE_COLUMNS = [
+  {
+    template: 'custom/composite-animalies-table/start-duration',
+    propertyName: 'startDuration',
+    title: `Start / Duration (${moment.tz([2012, 5], config.timeZone).format('z')})`
+  },
+  {
+    template: 'custom/composite-animalies-table/anomalies-list',
+    propertyName: 'details',
+    title: 'Anomalies details'
+  },
+  {
+    component: 'custom/composite-animalies-table/resolution',
+    propertyName: 'feedback',
+    title: 'Feedback '
+  }
+];
+
+export default Component.extend({
+  data: EmberArray(),
+  tagName: 'section',
+  customClasses: {
+    table: 'composite-anomalies-table'
+  },
+  title: 'Composite Anomalies', // Default Header if no title is passed in.
+  noRecords: 'No Composite Anomalies found',
+  tableData: computed('data', function () {
+    let computedTableData = [];
```

Comment:
This should be a `const` so that it is not reassigned to something else unintentionally.